### PR TITLE
chore: Change segment event name to generic version

### PR DESCRIPTION
**Pull request by Tj-Tracy**
**Description:**
Describe in a couple of sentences what this PR adds

**Merge checklist:**
- [ ] Bump version
- [ ] Add to changelog
- [ ] Update docs (not only docstrings)

**Post merge:**
(Will eventually be handled via automation)
- [ ] Release with new tag matching version

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

File: notices/static/notices/js/utils.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ function sendAcknowledgment(type, url){
   postRequest.onreadystatechange = () => {
     if (postRequest.readyState === 4 && postRequest.status === 204) {
       console.log("acknowledgment successful");
-      window.analytics.track('edx.bi.user.acknowledged_acqusition', {acknowledgment_type: type});
+      window.analytics.track('edx.bi.user.acknowledged_notice', {acknowledgment_type: type});
```
Comment (justinhynes) [inline notices/static/notices/js/utils.js:37]: Is there anything else we can add to the event we're emitting? Like, say, the notice id? So we know what notice this event is tied to?

Comment (Tj-Tracy) [reply notices/static/notices/js/utils.js]: Updated it so that any params that get sent to the POST also get sent with the segment event.

Comment (justinhynes) [reply notices/static/notices/js/utils.js]: Awesome, thanks!

**Review by justinhynes — APPROVED** [review]
LGTM!

**Pull request merged**